pulp_rpm/tests/functional/constants.py

```diff
@@ -126,6 +126,9 @@
 "content_summary" field on "../repositories/../versions/../".
 """
 
+RPM_UNSIGNED_FIXTURE_SIZE = 79260
```

Comment:
Are these numbers manually verified?

Reply:
Yes, I manually added up the sizes on the fixture. I then double checked by performing a sync on an empty system and adding up all the artifact sizes.